lighthouse-core/scripts/release/prepare-commit.sh

```diff
@@ -18,12 +18,6 @@ fi
 OLD_VERSION=$(node -e "console.log(require('./package.json').version)")
 NEW_VERSION=$1
 BRANCH_NAME="bump_$NEW_VERSION"
-SEMVER_PATTERN="[0-9]*\.[0-9]*\.[0-9]*"
-
-if [[ $(echo "$NEW_VERSION" | sed 's/[0-9]*\.[0-9]*\.[0-9]*/SECRET_REPLACE/g') != "SECRET_REPLACE" ]]; then
- echo "Incorrect version format. Must be x.x.x"
```

Comment:
can we still keep *some* sort of check here? :)

Reply:
wait do we even use this?

Reply:
the bump version script does the check now.